clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -986,6 +986,12 @@ private long sendEligibleCalls(long now) {
                     long nodeTimeout = client.pollDelayMs(node, now);
                     pollTimeout = Math.min(pollTimeout, nodeTimeout);
                     log.trace("Client is not ready to send to {}. Must delay {} ms", node, nodeTimeout);
+
+                    AuthenticationException authenticationException = client.authenticationException(node);
+                    if (authenticationException != null) {
```

Comment:
Do we have tests for this?

Reply:
I don't think this was strictly necessary. I added it during one of the refactors. I will remove it.